app/views/layouts/application.html.slim

```diff
@@ -29,7 +29,7 @@ html dir="ltr" lang="en-GB" class="govuk-template"
 
     = stylesheet_link_tag "application", "data-turbo-track": "reload"
     = javascript_include_tag "application", "data-turbo-track": "reload", defer: true, nonce: true
-  body class="govuk-template__body"
+  body class="govuk-template__body" data-sentry-dsn=ENV["SENTRY_CLIENT_SIDE_DSN"]
```

Comment:
What's the reason that we are creating a new variable `SENTRY_CLIENT_SIDE_DSN` instead of using the existing `SENTRY_DSN`?

Reply:
I'm now wondering if this is something to do with my above comment re. feature flag!

Reply:
So these are 2 different Sentry keys, one for server-side and one for client-side. I guess the reason is one is a private secret and one is public (because for it to be available to the JS client-side it needs to be available in the source of the page).

Reply:
As discussed on Slack, as you correctly pointed out, these are _not_ two separate keys - Sentry provides the same value for client-side and server-side DSNs. So we don't need to store it in 2 variables.